.devcontainer/devcontainer.json

```
@@ -39,7 +39,7 @@
     }
   ],
   "postCreateCommand": {
-    "set-ownership": "sudo chown vscode target",
+    "set-ownership": "sudo chown vscode target; sudo chown vscode /usr/local/cargo/registry/",
```

This should work too.

```suggestion
    "set-ownership": "sudo chown vscode target /usr/local/cargo/registry/",
```